app/assets/stylesheets/provider/_colors.scss

```diff
@@ -4,13 +4,15 @@
 // based on color names with main color attached for clarity if needed http://chir.ag/projects/name-that-color/
 
 // greys
-$white: #fff;
 $alabasterWhite: #fafafa; // pf-black-100
-$wildSandGray:   #f5f5f5; // pf-black-150
 $altoGray:       #d1d1d1; // pf-black-300
+$black:          #030303; // pf-black
 $osloGray:       #8b8d8f; // pf-black-500
 $outerSpaceGray: #393f44; // pf-black-800
-$black:          #030303; // pf-black
+$pfBlack200:     #ededed; // pf-black-200
```

Comment:
Let's stick with approximate css names for now: "Gallery" in this case. http://chir.ag/projects/name-that-color/#EDEDED